server/routes/cohortRoutes.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
```

Comment:
You are mixing cjs and esm import across these files. I would keep with esm eg:
`import express from "express";`
and
`export { router }`

Reply:
thank you to remind me , I changed all to the esm